### PR TITLE
Adding initial contents

### PR DESCRIPTION
**Pull request by jsignell**
Looks like:

Screen Shot 2019-06-05 at 5 13 54 PM
Screen Shot 2019-06-05 at 5 14 05 PM
Screen Shot 2019-06-05 at 5 14 15 PM

### CODE REVIEW

File: doc/index.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+Welcome to PyViz!  PyViz is a coordinated effort to make data visualization in Python easier to use, easier to learn, and more powerful.
+
+Focusing on interactive plotting in web browsers, PyViz contains:
+
+ - [Overviews](overviews/index.html) of the visualization landscape in Python and core concepts that many APIs build on.
+ - [Tutorials](tutorials/index.html) showing how to use the available tools together to do a wide range of different tasks.
+ - [Topics](htts://examples.pyviz.org) presenting narrative examples of generating visualizations for particular topics.
+
+## This site
+We are actively building this site and are looking for feedback/ideas/contributions.
```
Comment (jbednar) [inline doc/index.md]:
```suggestion
If you are a part of the Python data visualization landscape, then this is _your_ site!  All content has been contributed by individual library authors and users, and you could be next!
```

File: doc/index.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+So if you are a maintainer, contributor, or power-user of a
```
Comment (jbednar) [inline doc/index.md]:
```suggestion
```

File: doc/index.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+python visualization tool, we want to hear from you! Please open an issue or
```
Comment (jbednar) [inline doc/index.md]:
```suggestion
Please open an issue or
```

File: doc/index.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+PR on [this GitHub repo](https://github.com/pyviz/pyviz) to start a conversation.
+
+**NOTE:** The material that used to be hosted at https://pyviz.org can now be found at https://metaplot.org.
```
Comment (jbednar) [inline doc/index.md]:
```suggestion
**NOTE:** Some of the material previously hosted under Tutorials on this site is now hosted by metaplot.org and is linked from the "Metaplot" section of the Tutorials heading on this site.
```

File: doc/overviews/index.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
+# Overviews
+
+The Python Visualization Landscape can seem daunting at first. These
+overviews attempt to shine light on common patterns and use cases.
```
Comment (jbednar) [inline doc/overviews/index.md]: Overviews still to come, I assume?

Comment (jsignell) [reply doc/overviews/index.md]: They are in the side bar. I wasn't sure if links should be linked from the sidebar, from the main page or both.

**Comment by jsignell** [comment]
Some questions to consider: 
 - What content do we imagine being hosted on this site? Maybe this is just a kind of map that points to material in other places. In that case we might not need to render viz-heavy notebooks. 
 - Do we want to embed content (unified look and feel), or just link to external content (heavier maintenance burden)? 
 - Should links just be in text, just in nav, both?
 - Do we want to write markdown with an rst file that embeds that content (might be confusing for people)? Or do we just want to write markdown files and have them be rendered straight to html (less flexible. How do toctrees work?)? 
 - Should we be linking to other people's tutorials without asking?

File: doc/overviews/index.md (hunk 5 above)
Comment (jbednar) [reply doc/overviews/index.md]: Always both.

**Comment by jbednar** [comment]
>  - What content do we imagine being hosted on this site? Maybe this is just a kind of map that points to material in other places. In that case we might not need to render viz-heavy notebooks. 

I don't have any immediate need to host content directly on this site, but those plans could change depending on what people want to do, so I don't think there's a reason to disable the notebook support.

>  - Do we want to embed content (unified look and feel), or just link to external content (heavier maintenance burden)? 

I think that's a case by case basis.  We could achieve unified look and feel up to a point, but eventually if enough "foreign" content is linked to, we'll never be able to assimilate it all.

> - Should links just be in text, just in nav, both?

Always both, unless there is some very good reason.  Nav is for handy reference, text is for a short description; nav should capture and collect what's in the text, but in a list.

 - Do we want to write markdown with an rst file that embeds that content (might be confusing for people)? Or do we just want to write markdown files and have them be rendered straight to html (less flexible. How do toctrees work?)? 

Preferably .md where possible, for consistency with Github/Jupyter/how reasonable people think.  But somethings are only expressible in .rst, so we have to fall back to it.

 - Should we be linking to other people's tutorials without asking? 

I think we should link freely, but give people a heads up that the material is being linked in this way before we publicize anything.

**Pull request merged**